cpp/include/cudf/detail/gather.cuh

```diff
@@ -50,24 +51,24 @@ struct bounds_checker {
 template <bool ignore_out_of_bounds, typename MapIterator>
 __global__ void gather_bitmask_kernel(table_device_view source_table,
                                       MapIterator gather_map,
-                                      mutable_table_device_view destination_table,
+                                      bitmask_type * masks[],
```

Comment:
Why this change?

Reply:
String column can't have mutable device view, so just passing mask.